blobstore/wants.go

```diff
@@ -136,14 +165,29 @@ type wantProc struct {
 }
 
 func (proc *wantProc) init() {
+
+	proc.wmgr.promGauge("proc", 1)
+
 	bsCancel := proc.bs.Changes().Register(
 		luigi.FuncSink(func(ctx context.Context, v interface{}, err error) error {
 			proc.l.Lock()
 			defer proc.l.Unlock()
+			if luigi.IsEOS(err) {
```

Comment:
despite this one case I'm not sure what that input error is about, to be honest.

Reply:
The input error is EOS when `Close()` is called and `someErr` when `CloseWithError(someErr)` is called. So if `err` is neither EOS nor `nil`, it would be nice to log the error. If `err` is `nil`, `v` is not `nil` except someone poured a `nil`. Does that make sense?

Reply:
done